ionic_user_interface/src/components/VoteButton.vue

```diff
@@ -117,13 +128,19 @@ export default defineComponent({
   justify-content: center;
   font-size: 16px;
   border-radius: 10px;
-  border: 2px solid grey;
+  border: 2px solid var(--ion-color-danger);
   padding: 5px 7px;
   align-self: center;
 }
 
+.vote-button-component:hover {
+  background-color: var(--ion-color-danger-tint);
```

Comment:
![image](https://user-images.githubusercontent.com/46164638/126266055-85b759f1-936c-43e8-8cf2-84be3ed9fb7f.png)
The tint is too light and blends in with the bg, maybe try a slightly darker color? Alternatively only tint the heart itself